src/framework/components/element/image-element.js

```diff
@@ -593,6 +597,10 @@ pc.extend(pc, function () {
         set: function (value) {
             this._color.data[3] = value;
             this._meshInstance.setParameter("material_opacity", value);
+
+            if (this._element && this._element.fire) {
```

Comment:
Do you need to check if `this._element.fire` exists?

Reply:
Not really I guess, will remove.